### ISSUE TITLE
Event captured in before send but not sent to sentry dashboard

### ISSUE THREAD

**Issue by sreedharbukya** [getsentry/sentry-python#399]
I am using the latest version sentry-SDK with python3. 

 I have configured my sentry as follows in settings.py 

```python
def before_send(event, hint):
	log.info("Before Sending the Event, {} and Hint : {}".format(repr(event), repr(hint)))
	return event

def init_sentry():
	sentry_sdk.init(dsn=SENTRY_DSN, environment=ENVIRONMENT,
					attach_stacktrace=True, before_send=before_send)
	log.info("Sentry Initialized")
```
When some event got captured. I am able to check in log file for event in before_send.
But not received in actual sentry dashboard.

**Comment by untitaker** [comment]
Could you pass `debug=True` to show if sending the event causes any errors?

**Comment by sreedharbukya** [comment]
Thanks @untitaker , I got response 

```4
 [sentry] DEBUG: 0 event(s) pending on flush
Sentry is attempting to send 0 pending error messages
Waiting up to 2.0 seconds
Press Ctrl-C to quit
 [sentry] DEBUG: background worker flushed
 [sentry] DEBUG: Killing HTTP transport
 [sentry] DEBUG: background worker got kill request```

**Comment by ansizov** [comment]
I have the same problem.  If set invalid DSN, sentry client tries to send one event while shutdown timeout does not end.

```python
import logging
import sentry_sdk
from sentry_sdk.integrations.logging import LoggingIntegration

DSN = 'https://xxx@mysite.com/123'

sentry_sdk.init(
    dsn=DSN,
    integrations=[LoggingIntegration(
        level=logging.INFO,
        event_level=logging.INFO
    )],
    shutdown_timeout=5,
    debug=True
)

logging.error('some error')
```

Execute `python cli.py 2>&1 | tee log`

```
 [sentry] DEBUG: Setting up integrations (with default = True)
 [sentry] DEBUG: Setting up previously not enabled integration logging
 [sentry] DEBUG: Setting up previously not enabled integration stdlib
 [sentry] DEBUG: Setting up previously not enabled integration excepthook
 [sentry] DEBUG: Setting up previously not enabled integration dedupe
 [sentry] DEBUG: Setting up previously not enabled integration atexit
 [sentry] DEBUG: Setting up previously not enabled integration modules
 [sentry] DEBUG: Setting up previously not enabled integration argv
 [sentry] DEBUG: Setting up previously not enabled integration threading
 [sentry] DEBUG: Enabling integration logging
 [sentry] DEBUG: Enabling integration stdlib
 [sentry] DEBUG: Enabling integration excepthook
 [sentry] DEBUG: Enabling integration dedupe
 [sentry] DEBUG: Enabling integration atexit
 [sentry] DEBUG: Enabling integration modules
 [sentry] DEBUG: Enabling integration argv
 [sentry] DEBUG: Enabling integration threading
ERROR:root:some error
 [sentry] DEBUG: atexit: got shutdown signal
DEBUG:sentry_sdk.errors:atexit: got shutdown signal
 [sentry] DEBUG: Sending event, type:null level:error event_id:45c1a32d722d49558f6b0259cb63a129 project:123 host:mysite.com
DEBUG:sentry_sdk.errors:Sending event, type:null level:error event_id:45c1a32d722d49558f6b0259cb63a129 project:123 host:mysite.com
 [sentry] DEBUG: atexit: shutting down client
DEBUG:sentry_sdk.errors:atexit: shutting down client
 [sentry] DEBUG: Flushing HTTP transport
DEBUG:sentry_sdk.errors:Flushing HTTP transport
 [sentry] DEBUG: background worker got flush request
DEBUG:sentry_sdk.errors:background worker got flush request
 [sentry] DEBUG: 0 event(s) pending on flush
DEBUG:sentry_sdk.errors:0 event(s) pending on flush
Sentry is attempting to send 0 pending error messages
Waiting up to 5 seconds
Press Ctrl-C to quit
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7e2a8f10>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/123/store/
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c3a73d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/123/store/
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c3a7a10>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/123/store/
 [sentry] ERROR: Internal error in sentry_sdk
Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 157, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 672, in urlopen
    chunked=chunked,
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 376, in _make_request
    self._validate_conn(conn)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 994, in _validate_conn
    conn.connect()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 300, in connect
    conn = self._new_conn()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 169, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7cedc2d0>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 296, in send_event_wrapper
    self._send_event(event)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 217, in _send_event
    headers={"Content-Type": "application/json", "Content-Encoding": "gzip"},
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 164, in _send_request
    "POST", str(self._auth.store_api_url), body=body, headers=headers
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 80, in request
    method, url, fields=fields, headers=headers, **urlopen_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 171, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/poolmanager.py", line 330, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 720, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='mysite.com', port=443): Max retries exceeded with url: /api/123/store/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7cedc2d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:sentry_sdk.errors:Internal error in sentry_sdk
Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 157, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 672, in urlopen
    chunked=chunked,
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 376, in _make_request
    self._validate_conn(conn)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 994, in _validate_conn
    conn.connect()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 300, in connect
    conn = self._new_conn()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 169, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7cedc2d0>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 296, in send_event_wrapper
    self._send_event(event)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 217, in _send_event
    headers={"Content-Type": "application/json", "Content-Encoding": "gzip"},
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 164, in _send_request
    "POST", str(self._auth.store_api_url), body=body, headers=headers
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 80, in request
    method, url, fields=fields, headers=headers, **urlopen_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 171, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/poolmanager.py", line 330, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 720, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='mysite.com', port=443): Max retries exceeded with url: /api/123/store/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7cedc2d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
 [sentry] DEBUG: Sending event, type:null level:warning event_id:0c03c8eac9e94cf7a81b1010b43a2568 project:123 host:mysite.com
DEBUG:sentry_sdk.errors:Sending event, type:null level:warning event_id:0c03c8eac9e94cf7a81b1010b43a2568 project:123 host:mysite.com
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7cc017d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/123/store/
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c15d090>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/123/store/
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c15d6d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/123/store/
 [sentry] ERROR: Internal error in sentry_sdk
Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 157, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 672, in urlopen
    chunked=chunked,
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 376, in _make_request
    self._validate_conn(conn)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 994, in _validate_conn
    conn.connect()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 300, in connect
    conn = self._new_conn()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 169, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c15dd10>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 296, in send_event_wrapper
    self._send_event(event)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 217, in _send_event
    headers={"Content-Type": "application/json", "Content-Encoding": "gzip"},
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 164, in _send_request
    "POST", str(self._auth.store_api_url), body=body, headers=headers
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 80, in request
    method, url, fields=fields, headers=headers, **urlopen_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 171, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/poolmanager.py", line 330, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 720, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='mysite.com', port=443): Max retries exceeded with url: /api/123/store/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c15dd10>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:sentry_sdk.errors:Internal error in sentry_sdk
Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 157, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 672, in urlopen
    chunked=chunked,
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 376, in _make_request
    self._validate_conn(conn)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 994, in _validate_conn
    conn.connect()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 300, in connect
    conn = self._new_conn()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 169, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c15dd10>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 296, in send_event_wrapper
    self._send_event(event)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 217, in _send_event
    headers={"Content-Type": "application/json", "Content-Encoding": "gzip"},
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 164, in _send_request
    "POST", str(self._auth.store_api_url), body=body, headers=headers
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 80, in request
    method, url, fields=fields, headers=headers, **urlopen_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 171, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/poolmanager.py", line 330, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 720, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='mysite.com', port=443): Max retries exceeded with url: /api/123/store/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c15dd10>: Failed to establish a new connection: [Errno 111] Connection refused'))
 [sentry] DEBUG: Sending event, type:null level:warning event_id:7a1e295e227c4e498ed69390b90979d4 project:123 host:mysite.com
DEBUG:sentry_sdk.errors:Sending event, type:null level:warning event_id:7a1e295e227c4e498ed69390b90979d4 project:123 host:mysite.com
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c15d150>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/123/store/
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c16b550>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/123/store/
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c16bc50>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/123/store/
 [sentry] ERROR: Internal error in sentry_sdk
Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 157, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 672, in urlopen
    chunked=chunked,
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 376, in _make_request
    self._validate_conn(conn)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 994, in _validate_conn
    conn.connect()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 300, in connect
    conn = self._new_conn()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 169, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c176350>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 296, in send_event_wrapper
    self._send_event(event)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 217, in _send_event
    headers={"Content-Type": "application/json", "Content-Encoding": "gzip"},
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 164, in _send_request
    "POST", str(self._auth.store_api_url), body=body, headers=headers
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 80, in request
    method, url, fields=fields, headers=headers, **urlopen_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 171, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/poolmanager.py", line 330, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 720, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='mysite.com', port=443): Max retries exceeded with url: /api/123/store/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c176350>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:sentry_sdk.errors:Internal error in sentry_sdk
Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 157, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 672, in urlopen
    chunked=chunked,
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 376, in _make_request
    self._validate_conn(conn)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 994, in _validate_conn
    conn.connect()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 300, in connect
    conn = self._new_conn()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 169, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c176350>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 296, in send_event_wrapper
    self._send_event(event)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 217, in _send_event
    headers={"Content-Type": "application/json", "Content-Encoding": "gzip"},
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 164, in _send_request
    "POST", str(self._auth.store_api_url), body=body, headers=headers
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 80, in request
    method, url, fields=fields, headers=headers, **urlopen_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 171, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/poolmanager.py", line 330, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 720, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='mysite.com', port=443): Max retries exceeded with url: /api/123/store/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c176350>: Failed to establish a new connection: [Errno 111] Connection refused'))
 [sentry] DEBUG: Sending event, type:null level:warning event_id:1cca83a5f860481e8a43119342b2e14b project:123 host:mysite.com
DEBUG:sentry_sdk.errors:Sending event, type:null level:warning event_id:1cca83a5f860481e8a43119342b2e14b project:123 host:mysite.com
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c176510>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/123/store/
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c15d150>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/123/store/
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c15d350>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/123/store/
 [sentry] ERROR: Internal error in sentry_sdk
Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 157, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 672, in urlopen
    chunked=chunked,
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 376, in _make_request
    self._validate_conn(conn)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 994, in _validate_conn
    conn.connect()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 300, in connect
    conn = self._new_conn()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 169, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c155ed0>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 296, in send_event_wrapper
    self._send_event(event)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 217, in _send_event
    headers={"Content-Type": "application/json", "Content-Encoding": "gzip"},
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 164, in _send_request
    "POST", str(self._auth.store_api_url), body=body, headers=headers
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 80, in request
    method, url, fields=fields, headers=headers, **urlopen_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 171, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/poolmanager.py", line 330, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 720, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='mysite.com', port=443): Max retries exceeded with url: /api/123/store/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c155ed0>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:sentry_sdk.errors:Internal error in sentry_sdk
Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 157, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 672, in urlopen
    chunked=chunked,
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 376, in _make_request
    self._validate_conn(conn)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 994, in _validate_conn
    conn.connect()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 300, in connect
    conn = self._new_conn()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 169, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c155ed0>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 296, in send_event_wrapper
    self._send_event(event)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 217, in _send_event
    headers={"Content-Type": "application/json", "Content-Encoding": "gzip"},
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 164, in _send_request
    "POST", str(self._auth.store_api_url), body=body, headers=headers
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 80, in request
    method, url, fields=fields, headers=headers, **urlopen_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 171, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/poolmanager.py", line 330, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 720, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='mysite.com', port=443): Max retries exceeded with url: /api/123/store/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c155ed0>: Failed to establish a new connection: [Errno 111] Connection refused'))
 [sentry] DEBUG: Sending event, type:null level:warning event_id:f046af0775f8455aa45b0e88e2a73d71 project:123 host:mysite.com
DEBUG:sentry_sdk.errors:Sending event, type:null level:warning event_id:f046af0775f8455aa45b0e88e2a73d71 project:123 host:mysite.com
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7cc017d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/123/store/
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c176350>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/123/store/
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c15d450>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/123/store/
 [sentry] ERROR: Internal error in sentry_sdk
Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 157, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 672, in urlopen
    chunked=chunked,
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 376, in _make_request
    self._validate_conn(conn)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 994, in _validate_conn
    conn.connect()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 300, in connect
    conn = self._new_conn()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 169, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c16b910>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 296, in send_event_wrapper
    self._send_event(event)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 217, in _send_event
    headers={"Content-Type": "application/json", "Content-Encoding": "gzip"},
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 164, in _send_request
    "POST", str(self._auth.store_api_url), body=body, headers=headers
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 80, in request
    method, url, fields=fields, headers=headers, **urlopen_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 171, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/poolmanager.py", line 330, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 720, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='mysite.com', port=443): Max retries exceeded with url: /api/123/store/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c16b910>: Failed to establish a new connection: [Errno 111] Connection refused'))
ERROR:sentry_sdk.errors:Internal error in sentry_sdk
Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 157, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 672, in urlopen
    chunked=chunked,
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 376, in _make_request
    self._validate_conn(conn)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 994, in _validate_conn
    conn.connect()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 300, in connect
    conn = self._new_conn()
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connection.py", line 169, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c16b910>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 296, in send_event_wrapper
    self._send_event(event)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 217, in _send_event
    headers={"Content-Type": "application/json", "Content-Encoding": "gzip"},
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/sentry_sdk/transport.py", line 164, in _send_request
    "POST", str(self._auth.store_api_url), body=body, headers=headers
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 80, in request
    method, url, fields=fields, headers=headers, **urlopen_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/request.py", line 171, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/poolmanager.py", line 330, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 760, in urlopen
    **response_kw
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/connectionpool.py", line 720, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/tmp/sentry/.venv/lib/python3.7/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='mysite.com', port=443): Max retries exceeded with url: /api/123/store/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f2c7c16b910>: Failed to establish a new connection: [Errno 111] Connection refused'))
 [sentry] DEBUG: Sending event, type:null level:warning event_id:1a3e309ec6ab48e69f97306f6ea632dd project:123 host:mysite.com
DEBUG:sentry_sdk.errors:Sending event, type:null level:warning event_id:1a3e309ec6ab48e69f97306f6ea632dd project:123 host:mysite.com
 [sentry] DEBUG: background worker flushed
DEBUG:sentry_sdk.errors:background worker flushed
 [sentry] DEBUG: Killing HTTP transport
DEBUG:sentry_sdk.errors:Killing HTTP transport
 [sentry] DEBUG: background worker got kill request
DEBUG:sentry_sdk.errors:background worker got kill request
```

- python3.7
- sentry-sdk==0.14.3
- urllib3==1.25.8

**Comment by untitaker** [comment]
@ansizov that seems entirely unrelated to OP's problem. Could you open a new issue?

**Comment by Nihisil** [comment]
I have same issue, is there any way to debug it further? Events working fine locally, but from production I can't get sentry events.

OS is alpine, Python 3.9, sentry-sdk==1.0.0.

Log with debug enabled:
```
/app # python scripts/create_user.py 1 2 3
 [sentry] DEBUG: Setting up integrations (with default = True)
 [sentry] DEBUG: Did not import default integration sentry_sdk.integrations.django.DjangoIntegration: Django not installed
 [sentry] DEBUG: Did not import default integration sentry_sdk.integrations.flask.FlaskIntegration: Flask is not installed
 [sentry] DEBUG: Did not import default integration sentry_sdk.integrations.bottle.BottleIntegration: Bottle not installed
 [sentry] DEBUG: Did not import default integration sentry_sdk.integrations.falcon.FalconIntegration: Falcon not installed
 [sentry] DEBUG: Did not import default integration sentry_sdk.integrations.sanic.SanicIntegration: Sanic not installed
 [sentry] DEBUG: Did not import default integration sentry_sdk.integrations.celery.CeleryIntegration: Celery not installed
 [sentry] DEBUG: Did not import default integration sentry_sdk.integrations.rq.RqIntegration: RQ not installed
 [sentry] DEBUG: Did not import default integration sentry_sdk.integrations.aiohttp.AioHttpIntegration: AIOHTTP not installed
 [sentry] DEBUG: Did not import default integration sentry_sdk.integrations.tornado.TornadoIntegration: Tornado not installed
 [sentry] DEBUG: Did not import default integration sentry_sdk.integrations.sqlalchemy.SqlalchemyIntegration: SQLAlchemy not installed.
 [sentry] DEBUG: Did not import default integration sentry_sdk.integrations.boto3.Boto3Integration: botocore is not installed
 [sentry] DEBUG: Setting up previously not enabled integration logging
 [sentry] DEBUG: Setting up previously not enabled integration stdlib
 [sentry] DEBUG: Setting up previously not enabled integration excepthook
 [sentry] DEBUG: Setting up previously not enabled integration dedupe
 [sentry] DEBUG: Setting up previously not enabled integration atexit
 [sentry] DEBUG: Setting up previously not enabled integration modules
 [sentry] DEBUG: Setting up previously not enabled integration argv
 [sentry] DEBUG: Setting up previously not enabled integration threading
 [sentry] DEBUG: Enabling integration logging
 [sentry] DEBUG: Enabling integration stdlib
 [sentry] DEBUG: Enabling integration excepthook
 [sentry] DEBUG: Enabling integration dedupe
 [sentry] DEBUG: Enabling integration atexit
 [sentry] DEBUG: Enabling integration modules
 [sentry] DEBUG: Enabling integration argv
 [sentry] DEBUG: Enabling integration threading
Traceback (most recent call last):
  File "/app/scripts/create_user.py", line 24, in <module>
 [sentry] DEBUG: Sending event, type:null level:error event_id:* project:* *.sentry.io
    main()
  File "/app/scripts/create_user.py", line 17, in main
    b = 123 / 0
ZeroDivisionError: division by zero
 [sentry] DEBUG: atexit: got shutdown signal
 [sentry] DEBUG: atexit: shutting down client
 [sentry] DEBUG: Flushing HTTP transport
 [sentry] DEBUG: background worker got flush request
 [sentry] DEBUG: 1 event(s) pending on flush
Sentry is attempting to send 1 pending error messages
Waiting up to 2 seconds
Press Ctrl-C to quit
 [sentry] DEBUG: background worker flushed
 [sentry] DEBUG: Killing HTTP transport
 [sentry] DEBUG: background worker got kill request
```

**Comment by antonpirker** [comment]
As the OP has never replied I guess the problem was solved or it not relevant anymore. I am closing this.
